components/sanity/scripts/start.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+
+import os
+import json
+import requests
+from os import remove
+from os.path import join, dirname, isfile
+
+from cloudify import ctx
+
+ctx.download_resource(
+        join('components', 'utils.py'),
+        join(dirname(__file__), 'utils.py'))
+import utils  # NOQA
+
+BLUEPRINT_ID = 'sanity_bp'
+DEPLOYMENT_ID = 'sanity_deployment'
+SANITY_SERVICE_NAME = 'sanity'
+
+manager_ip = os.environ.get('manager_ip')
+manager_user = ctx.instance.runtime_properties['manager_user']
+manager_remote_key_path = \
+    ctx.instance.runtime_properties['manager_remote_key_path']
+
+ctx_properties = utils.ctx_factory.create(SANITY_SERVICE_NAME)
+
+
+def _prepare_sanity_app():
+    sanity_app_source_url = ctx_properties['sanity_app_source_url']
+    app_tar = utils.download_cloudify_resource(
+              url=sanity_app_source_url,
+              service_name=SANITY_SERVICE_NAME)
+
+    ctx.instance.runtime_properties['sanity_app_tar'] = app_tar
```

Comment:
do you need all of these other than just for passing them between functions? If so.. updating the object is more process heavy than passing them as arguments.